platform/core.startup/src/org/netbeans/core/startup/ConsistencyVerifier.java

```diff
@@ -152,7 +152,7 @@ private ConsistencyVerifier() {}
             for (Object prob : probs) {
                 if (prob instanceof Dependency) {
                     Dependency d = (Dependency) prob;
-                    if (Dependency.TYPE_JAVA == d.getType() && m.isAutoload()) {
+                    if (Dependency.TYPE_JAVA == d.getType() && (m.isEager() || m.isAutoload())) {
```

Comment:
Ignnores unsatisfied Java dependencies for both `eager` and `autoload` modules: when tests run on JDK8 dependencies on JDK9/11 makes the module to fail. If it is eager, it's OK to ignore, it's just enhancing something.